### PR TITLE
Update queries conforming to latest nvim-treesitter

### PR DESCRIPTION
**Pull request by jayadamsmorgan**
Updated versions of queries for nvim-treesitter (#2). 

tree-sitter-pkl version in `internal.lua` had to be updated to support opening square bracket capture.
Highlights and indents are still not perfect but it's a start.

### CODE REVIEW

**Comment by jayadamsmorgan** [comment]
Users who already installed this plugin will have to re-run `:TSInstall pkl' in order to update tree-sitter-pkl and make highlights work again. Not sure what to do about it.

File: queries/pkl/highlights.scm [hunk 1]
```diff
@@ -71,24 +67,24 @@
...
 (interpolationExpr
  "\\#(" @punctuation.special
- ")" @punctuation.special) @none
-  
+ ")" @punctuation.special) @embedded
+
 (interpolationExpr
   "\\##(" @punctuation.special
-  ")" @punctuation.special) @none
+  ")" @punctuation.special) @embedded
```
Comment (lucario387) [inline queries/pkl/highlights.scm:78]: What is this capture, Neovim doesn't have this capture

File: queries/pkl/highlights.scm [hunk 2]
```diff
@@ -132,21 +128,20 @@
...
-(importClause "import" @include)
-(importGlobClause "import*" @include)
-(importClause "as" @include)
-"in" @repeat
-"is" @keyword.operator
+"if" @keyword.conditional
+(importExpr "import" @module)
+(importGlobExpr "import*" @module)
+"import" @keyword.import
+"import*" @keyword.import
```
Comment (lucario387) [inline queries/pkl/highlights.scm:142]: Neovim used last-win order (and so is Helix and upstream Treesitter as of now). This made it so only keyword.import is shown

**Review by lucario387 — COMMENTED** [review]
Please follow nvim-treesitter's CONTRIBUTING.md for highlight captures.

File: queries/pkl/highlights.scm (hunk 1 above)
Comment (jayadamsmorgan) [reply queries/pkl/highlights.scm:78]: Thank you so much for your review, I will fix it.
Most of 'highlights.scm' commit contents were grabbed from upstream tree-sitter-pkl, and I thought this one was used for something.

File: queries/pkl/highlights.scm (hunk 2 above)
Comment (jayadamsmorgan) [reply queries/pkl/highlights.scm:142]: Can you please clarify this? Should I correct it to just `@keyword` instead of '@keyword.import' as it was before?

Comment (lucario387) [reply queries/pkl/highlights.scm:142]: If two captures to the same node, i.e

```
(Foo (a) @capture1)
(a) @capture2  
```

Then the capture appear last will be used for highlighting

Tl;dr: cascading highlights

**Comment by lucario387** [comment]
Me reviewing won't do much as I don't have merge privileges :)

Let's just say I approve the indent queries, highlights idk

**Review by bioball — APPROVED** [review]
Thank you!
